src/main/java/org/apache/sling/api/resource/external/package-info.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-@Version("1.0.2")
+@Version("1.1.0")
```

Comment:
With the minor version increase we break every URIProvider - which is not necessary as the new method has a default method

Reply:
Yes, but if we would only increase the patch version consumers could not depend on the new method easily due to the default import version range being used by Bnd (https://bnd.bndtools.org/chapters/170-versioning.html#import-version-policy). I want consumer bundles to no start which depend on this method, in case only 1.0.2 of this package is deployed!

Reply:
What about exporting this package with two versions? Do you know which version is picked by Bnd at build time to generate the import-version-policy?

Reply:
I think that's too complicated - so I fear we have to increase the minor version which is really very unfortunate